Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -71,11 +70,11 @@ public bool SetupAtRuntime
         }
 
         [SerializeField]
-        [Tooltip("Number of lines visible in scroller. Orthogonal to tiers.")]
+        [Tooltip("Number of visible tiers in the scrolling area.")]
```

Comment:
>tiers [](start = 36, length = 5)

Would it be possible to have something define the words Tier, Page, and Index?

These are fairly important in understanding the mechanics of how scrolling works (as seen by ScrollViewTests which do tier/page/index navigation).

I could write up some ASCII art if this would help :)

Reply:
Adding this to next graduation PR